### PR TITLE
`3.39.0` changelog

### PR DESCRIPTION
**Pull request by rileykarson**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 ## 3.39.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added `network` field to `compute_target_instance` ([#7189](https://github.com/hashicorp/terraform-provider-google/pull/7189))
+* compute: added `storage_locations` field to `google_compute_snapshot` ([#7201](https://github.com/hashicorp/terraform-provider-google/pull/7201))
+* container: added `self_link` to `google_container_cluster` ([#7191](https://github.com/hashicorp/terraform-provider-google/pull/7191))
+* container: marked `workload_metadata_config` as GA in `google_container_node_pool` ([#7192](https://github.com/hashicorp/terraform-provider-google/pull/7192))
+
+BUG FIXES:
+* bigquery: fixed a bug when a big query table schema didn't have `name` in the schema it'd panic, now logs an error. ([#7215](https://github.com/hashicorp/terraform-provider-google/pull/7215))
```
Comment (danawillow) [inline CHANGELOG.md:10]: same phrasing suggestion from the other one

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 ## 3.39.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added `network` field to `compute_target_instance` ([#7189](https://github.com/hashicorp/terraform-provider-google/pull/7189))
```
Comment (danawillow) [inline CHANGELOG.md:4]: looks like this is beta-only

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+* compute: added `storage_locations` field to `google_compute_snapshot` ([#7201](https://github.com/hashicorp/terraform-provider-google/pull/7201))
```
Comment (danawillow) [inline CHANGELOG.md:5]: this PR looks like it also does a few other things, can those get changelog notes too?

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,18 @@
 ## 3.39.0 (Unreleased)
 
+IMPROVEMENTS:
+* compute: added `storage_locations` field to `google_compute_snapshot` ([#7201](https://github.com/hashicorp/terraform-provider-google/pull/7201))
+* compute: added `kms_key_service_account`, `kms_key_self_link ` fields to `snapshot_encryption_key` field in `google_compute_snapshot`
+* compute: added `source_disk_encryption_key.kms_key_service_account` field to `google_compute_snapshot`
```
Comment (megan07) [inline CHANGELOG.md:6]: This entry and the one above it don't have the links to the PR after them - I think they should be the same as the first one, they were all done together.

**Review by megan07 — APPROVED** [review]

**Pull request merged**